lib/snowplow_tracker/emitters/server.ex

```diff
@@ -0,0 +1,51 @@
+defmodule SnowplowTracker.Emitters.Server do
+  @moduledoc """
+  This module defines the genserver that will be responsible for scheduling the jobs to 
+  fetch events from ETS
+  """
+
+  use GenServer
+
+  require Logger
+
+  alias SnowplowTracker.Emitters.{Processor, Cache}
+
+  def start_link(_args) do
+    GenServer.start_link(
+      __MODULE__,
+      nil,
+      name: __MODULE__
+    )
+  end
+
+  def init(_args) do
+    Cache.init()
```

Comment:
Move these tasks to `handle_continue/2`

https://hexdocs.pm/elixir/GenServer.html#c:handle_continue/2

Read more here - https://medium.com/@tylerpachal/introduction-to-handle-continue-in-elixir-and-when-to-use-it-53ba5519cc17